components/WebinarUpcomingState.tsx

```
@@ -0,0 +1,198 @@
+
+import { type WebinarUpcomingState } from "@/types";
+import CountdownTimer from "./webinar/CountdownTimer";
+import WaitlistComponent from "./webinar/WaitlistComponent";
+import { WebinarStatusEnum } from "@prisma/client";
+import { Button } from "./ui/button";
+import { useRouter } from "next/navigation";
+import { toast } from "sonner";
+import { changeWebinarStatus } from "@/actions/webinar.actions";
+import { Loader2, Calendar } from "lucide-react";
+
+const WebinarUpcomingState = ({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'WebinarUpcomingState'. Consider to delete it or rename it.

'WebinarUpcomingState' is defined here:


(lint/suspicious/noRedeclare)